src/Logger-DeviceClass_template/main.py

```diff
@@ -225,47 +225,47 @@ def initialize(self):
 
     def deinitialize(self):
         # called only once at the end of the measurement
-        debug("deinitialize")
-
-    def poweron(self):
-        # called if the measurement procedure enters a branch of the sequencer
-        # and the module has not been used in the previous branch
-        debug("poweron")
-
-    def poweroff(self):
-        # called if the measurement procedure leaves a branch of the sequencer
-        # and the module is not used in the next branch
-        debug("poweroff")
+        debug("-> deinitialize")
 
     def configure(self):
         # called if the measurement procedure enters a branch of the sequencer
         # and the module has not been used in the previous branch
-        debug("configure")
+        debug("->  configure")
 
     def unconfigure(self):
         # called if the measurement procedure leaves a branch of the sequencer
         # and the module is not used in the next branch
-        debug("unconfigure")
+        debug("->  unconfigure")
 
     def signin(self):
         # called if the variation of the module that is loading this device class starts
-        debug("signin")
+        debug("->   signin")
 
     def signout(self):
         # called if the variation of the module that is loading this device class ends
-        debug("signout")
-
+        debug("->   signout")
+        
     def reconfigure(self, parameters, keys):
         """'reconfigure' is called whenever parameters of the GUI change by using the {...}-parameter system."""
         debug()
-        debug("reconfigure")
-        debug("Parameters:", parameters)
-        debug("Changed keys:", keys)
+        debug("->  reconfigure")
+        debug("->  Parameters:", parameters)
+        debug("->  Changed keys:", keys)
 
         ### The following two lines are the default behavior that is used by EmptyDevice
```

Comment:
Wondering if we should include a comment saying that you can use `super().reconfigure()` if you want to override the method and at the same time still use the EmptyDevice behaviour.

On the other hand, that could be added to every function, so then why write it here.

Any opinions?

Reply:
use of super().... is not recommended for any of the semantic functions, so only experts should us it and I would not recommend it to beginners in the template. It would only make sense for reconfigure so far because all other functions just use pass in EmptyDevice.